Cargo.toml

```diff
@@ -44,3 +49,8 @@ rustdoc-args = ["--cfg", "docsrs"]
 [[bench]]
 name = "bench"
 harness = false
+
+[profile.bench]
```

Comment:
Was there a specific reason for this? `opt-level=3` I think is the default one, but you're overriding `lto` and `codegen-units`.

Reply:
It is indeed the default, I think I just copy&pasted the whole thing from somewhere. Removing.